thrift/src/main/thrift/atoms/chart.thrift

```diff
@@ -0,0 +1,83 @@
+namespace * contentatom.chart
+namespace java com.gu.contentatom.thrift.atom.chart
+
+enum ChartType {
+    BAR = 0
+    BARGROUP = 1
+    BARGROUPSTACK = 2
+    BARGROUPSTACK100 = 3
+    BAR100 = 4
+    BROKENBAR100 = 5
+    COL = 6
+    COLGROUP = 7
+    COLGROUPSTACK = 8
+    COLGROUPSTACK100 = 9
+    LINEDISCRETE = 10
+    LINECONTINUE = 11
+    PLOTDOT = 12
+    PLOTSCATTER = 13
+    ONBARDIFFARROW = 14
+    ONBARDIFFDOTS = 15
+    ONBARTICKS = 16
+    SLOPEGRAPH = 17
+}
+
+struct Range {
+    1: required double min
+    2: required double max
+}
+
+struct DisplaySettings {
+    1: required bool showHeadline = true
+    2: required bool showSource = true
+    3: optional bool showStandfirst
+    4: optional bool showLegend
+}
+
+struct Furniture {
+    1: required string headline
+    2: required string source
+    3: optional string standfirst
+}
+
+enum RowType {
+    STRING = 0,
+    DATE = 1
+}
+
+struct RowHeader {
```

Comment:
I think we need another enum for the formatting, i.e. regular, bold, italic, bold italic. What do you think?

Reply:
Currently, editors add these html tags to the `value`. e.g.

| name | age |
| -- | -- |
| foo | 1 |
| `<b>bar</b>` | 2 |

I don't think there's any restriction on the tags... so you could iframe in something 😱. Perhaps providing more structure is better. cc @susiecoleman

Reply:
Yeah, I remember in the demo at some point the styles got lost of wrong and he had to undo everything. If we decouple then this will never happen again.

Reply:
I like the idea of being more restrictive but I wonder if `London <b>UK</b>` (or something similar) is a common pattern